lib/cocoapods/target/aggregate_target.rb

```diff
@@ -191,6 +191,19 @@ def uses_swift?
       pod_targets.any?(&:uses_swift?)
     end
 
+    # @return [Boolean] Whether the target contains any resources
+    #
+    def has_resources?
+      !resource_paths_by_config.values.all?(&:empty?)
+    end
+
+    # @return [Boolean] Whether the target contains framework to be embedded into
+    #         the user target
+    #
+    def has_frameworks?
```

Comment:
rubocop doesn't like the name of these - any suggestions

Reply:
`includes_...?`